lms/envs/common.py

```diff
@@ -555,6 +555,7 @@
     OPENEDX_ROOT / 'core' / 'djangoapps' / 'dark_lang' / 'templates',
     OPENEDX_ROOT / 'core' / 'lib' / 'license' / 'templates',
     OPENEDX_ROOT / 'features' / 'course_experience' / 'templates',
+    OPENEDX_ROOT / 'features' / 'edly' / 'templates',
```

Comment:
Why we are doing this?

Reply:
I was trying to render the mako email template using edxmako and this library searches for templates only in the directories listed here.